docker-compose.yml

```diff
@@ -44,46 +44,91 @@ services:
     networks:
       - local
 
-  connectora:
-    image: ghcr.io/international-data-spaces-association/dataspace-connector:7.1.0
-    container_name: connectora
+# Add the PostgreSQL container to be used by DataspaceConnectorA and DataspaceConnectorB
```

Comment:
While adding postgres it might be useful to also add a Docker container for PGWEB so one can easily view the postgres instances. What do you think?